scripts/ef-tests/debug.py

```diff
@@ -64,7 +98,18 @@ def set_pre_state(w3, data):
         w3.provider.make_request("anvil_setBalance", [address, account["balance"]])
         w3.provider.make_request("anvil_setNonce", [address, account["nonce"]])
         for k, v in account["storage"].items():
-            w3.provider.make_request("anvil_setStorage", [address, k, v])
+            w3.provider.make_request(
+                "anvil_setStorageAt", [address, f"0x{k[2:]:0<64}", f"0x{v[2:]:0<64}"]
```

Comment:
At first I though that `k` and `v` were `int`; I think that you could put either

```
f"{k:0<66}"
```

or

```
k.ljust(66, "0")
```

Reply:
does this `f"{k:0<66}"` work? Because k and v are in the form "0x...."

Reply:
it does, you can try fr yourself